libs/crypto/tests/unit/mcl_dkg_tests.cpp

```diff
@@ -0,0 +1,271 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "crypto/mcl_dkg.hpp"
+#define MCLBN_FP_UNIT_SIZE 4
```

Comment:
I am not sure this define here will do anything if it happens after the MCL library has been parsed and included.

Reply:
Also, what does FP mean? Floating point?

Reply:
Fp is a class in the mcl library, which has a private member that is an array. You can correct that the define like doesn't doing anything and wasn't needed.